examples/openai_audio_api_client.py

```diff
@@ -0,0 +1,86 @@
+"""An example showing how to use vLLM to serve VLMs.
+
+Launch the vLLM server with the following command:
+vllm serve fixie-ai/ultravox-v0_3
+"""
+import base64
+
+import requests
+from openai import OpenAI
+
+# Modify OpenAI's API key and API base to use vLLM's API server.
+openai_api_key = "EMPTY"
+openai_api_base = "http://localhost:8000/v1"
+
+client = OpenAI(
+    # defaults to os.environ.get("OPENAI_API_KEY")
+    api_key=openai_api_key,
+    base_url=openai_api_base,
+)
+
+models = client.models.list()
+model = models.data[0].id
+
+audio_url = "https://upload.wikimedia.org/wikipedia/en/b/bf/Dave_Niehaus_Winning_Call_1995_AL_Division_Series.ogg"
```

Comment:
Does it accept `.ogg` files only? Perhaps make a comment here on the file format.

Reply:
> Are you able to upload to that bucket?

Yep I am, we can probably keep the example here as is though. The s3 bucket is mostly used for testing purposes, so we should probably add it to `vllm/assets` as an audio asset. I can do it in another PR

Reply:
Actually.. let me do that now since we haven't run the tests for this PR yet

Reply:
@petersalas I've uploaded two `.ogg` files to this bucket - do you mind creating an `audio.py` under `vllm/assets` similar to `image.py`? (Don't worry about the embeddings)

The file locations are
```
s3://vllm-public-assets/multimodal_asset/winning_call.ogg
s3://vllm-public-assets/multimodal_asset/mary_had_lamb.ogg
```
so it would be something like
```
    @property
    def audio_ndarray(self) -> np.ndarray:

        audio_path = get_vllm_public_assets(filename=f"{self.name}.ogg",
                                            s3_prefix="multimodal_asset")
        return librosa.load(audio_path, sr=None)
```

Reply:
Done, thanks for adding those!